js-extensions/@jenkins-cd/subs/extensions-bundle.js

```diff
@@ -162,3 +184,47 @@ function assertHasJenkinsJsExtensionsDependency(message) {
         dependencies.exitOnMissingDependency('@jenkins-cd/js-extensions', message);
     }
 }
+
+function findI18nBundles() {
+    var bundles = [];
+    var bundleFileDir = getPluginResourceBundleDir();
+
+    if (fs.existsSync(bundleFileDir)) {
+        var files = fs.readdirSync(bundleFileDir);
+        if (files) {
+            var cpPrefix = getPluginResourceBundleClasspath();
+            var propertiesFileMatcher = /\.properties$/; // endswith ".properties"
+            var everythingAfterFirstUnderscoreMatcher = /_.*$/;
+
+            for (var i = 0; i < files.length; i++) {
+                var file = files[i];
+                if (propertiesFileMatcher.test(file)) {
+                    // Strip off the different parts we don't want. It might be that
+                    // resource bundle basenames can have underscores in them according
+                    // to specs (I didn't find a ref one way or the other), but trying to match
+                    // language/country/variant codes exactly and strip them off is going to
+                    // make the code a pita to impl/understand, so lets just say we only detect
+                    // basenames that do NOT contain underscores. If someone insists on using
+                    // underscores they can manually define the bundles in the yaml file.
```

Comment:
Do I understood the comment correctly that we only need the basename. Maybe add to the above a small example:

```
cd blueocean-plugin/blueocean-dashboard/src/main/resources/jenkins/plugins/blueocean/dashboard
tree 
.
├── Messages_de.properties
├── Messages_es.properties
├── Messages_fr.properties
└── Messages.properties
```

In case of the dashboard plugin we have above resource bundles. The only file we are interested in in `Messages.properties` - the base name of the bundle - all other files are variants/translations